### PR TITLE
feat(firestore-stripe-payments): Allow configuring min instances for createCheckoutSession

### PR DESCRIPTION
**Pull request by abierbaum**
I am proposing a new configuration setting that would allow setting the minInstances for the createCheckouSession function.

The response time of this function is critical for how fast users percieve the checkout experience.  I have found times where the response can take >7-8 seconds and this amount of waiting leads to users starting the checkout process and then abandoning because they think the process is hung.  

This change would allow using firebase functions support for idle instances to ensure there are instances ready to process the request and ensure higher speed response for checkout.

see: https://firebase.google.com/docs/functions/manage-functions#min-max-instances

Note: I have not been able to test these changes as I do not know a way to test extensions.  I am putting up the PR as an example of how I think it _could_ work to discuss with the maintainers.

### CODE REVIEW

**Comment by abierbaum** [comment]
@jsteele-stripe @lemuel-stripe I am not sure if this is of interest to the project, but I think it would help me and likely other users tremendously with conversion improvements.  The current checkout flow simply takes too long from cold starts, atleast in my experience running it for several months.

File: firestore-stripe-payments/extension.yaml [hunk 1]
```diff
@@ -239,10 +239,21 @@ params:
...
       postinstall instructions for registering your webhook
       and configuring this value.
     example: whsec_1234567890
     required: false
+
+  - param: CREATE_CHECKOUT_MIN_INSTANCES
+    label: Minimum instances for create checkout handler
+    type: secret
+    description: >-
+      How many instances should be always ready to process checkout requests?
```
Comment (jsteele-stripe) [inline firestore-stripe-payments/extension.yaml:254]: I think it's important we highlight the potential Firebase cost implications of this configuration option. Can we add a callout and link to the [relevant](https://firebase.google.com/docs/functions/manage-functions#reduce_the_number_of_cold_starts) Firebase documentation here?

File: firestore-stripe-payments/extension.yaml [hunk 2]
```diff
@@ -239,10 +239,21 @@ params:
...
+      This is your signing secret for a Stripe-registered webhook.
       This webhook can only be registered after installation.
-      Leave this value untouched during installation, then follow the 
+      Leave this value untouched during installation, then follow the
       postinstall instructions for registering your webhook
       and configuring this value.
     example: whsec_1234567890
     required: false
+
+  - param: CREATE_CHECKOUT_MIN_INSTANCES
```
Comment (jsteele-stripe) [inline firestore-stripe-payments/extension.yaml:250]:
```suggestion
  - param: CREATE_CHECKOUT_SESSION_MIN_INSTANCES
```

File: firestore-stripe-payments/extension.yaml [hunk 3]
```diff
@@ -239,10 +239,21 @@ params:
...
+    label: Minimum instances for create checkout handler
```
Comment (jsteele-stripe) [inline firestore-stripe-payments/extension.yaml:251]:
```suggestion
    label: Minimum instances for createCheckoutSession function
```

**Review by jsteele-stripe — APPROVED** [review]
Minor comments to address, otherwise LGTM!

cc @dackers86 for Firebase context.

File: firestore-stripe-payments/extension.yaml (hunk 1 above)
Comment (jsteele-stripe) [reply firestore-stripe-payments/extension.yaml:254]: Unable to make this change directly as the PR is from a forked branch. Will merge and add extra context retrospectively.

Comment (jsteele-stripe) [reply firestore-stripe-payments/extension.yaml:254]: https://github.com/stripe/stripe-firebase-extensions/commit/b93e13aea473b76751078c3e01671a987631112d

**Pull request merged**